probe-rs/src/architecture/riscv/mod.rs

```diff
@@ -1,6 +1,7 @@
 //! RISCV Support
 
 #![allow(clippy::inconsistent_digit_grouping)]
+#![allow(clippy::unusual_byte_groupings)]
```

Comment:
This does not work somehow, is this because of a clippy version missmatch between your local install and the one on the CI?

Reply:
I tried this on macOS, where I had to use the beta version. So this will probably come back with the next Rust release :wink: